projects/aca-content/src/lib/components/search/search-input/search-input.component.spec.ts

```diff
@@ -59,6 +70,13 @@ describe('SearchInputComponent', () => {
     fixture.destroy();
   });
 
+  it('sidenav should be collapsed by default', () => {
```

Comment:
Comment for all tests: 
we have rule that all test titles should start with "should" word. In your case you start titles with "sidenav". Please rename all tests